packages/shared-lib/src/app/schema/easy-genomics/laboratory-run.ts

```diff
@@ -18,7 +18,7 @@ export const LaboratoryRunSchema = z
     ExternalRunId: z.string().optional(),
     S3Input: z.string().optional(),
     S3Output: z.string().optional(),
-    Settings: z.string().optional(), // JSON string
+    Settings: z.union([z.string(), z.record(z.string(), z.any())]).optional(), // JSON string
```

Comment:
@zigtan This was necessary in order to validate against both `{}` and `""`values currently being returned in the laboratory runs response.